app/containers/Scatter/saga.js

```diff
@@ -80,5 +107,5 @@ function* watchScatterRemove() {
 //
 
 export default function* rootSaga() {
-  yield all([watchScatterLoaded(), watchScatterConnect(), watchScatterRemove()]);
+  yield all([watchScatterLoaded(), watchScatterConnect(), watchScatterRemove(), watchEosRefreshData()]);
```

Comment:
Same here, `yarn run format:js` should fix it

Reply:
I swear I did...

Reply:
Haha same then, all good :)

You can try `prettier` very quickly by, say, replacing single quotes by double quotes in the code. You run `format:js` and you'll see the magic